themes/survey/fruity_twentythree/css/variations/theme_blueberry.css

```
@@ -15511,122 +15511,323 @@ body .imageselect-listitem label::after {
+  table.ls-answers .checkbox-item label.ls-label-xs-visibility::before {
+    margin-left: 0px;
+  }
+  table.ls-answers .checkbox-item label.ls-label-xs-visibility::after {
+    margin-left: 0px;
+  }
+}
+.success-border {
+  border-color: #5076FF;
+}
+
+@media screen and (max-width: 767px) {
```

Stylelint (17.2.0)

[error] Expected "context" media feature range notation (media-feature-range-notation)

(media-feature-range-notation)